src/main/java/com/patchworkmc/event/EventSubscriptionChecker.java

```diff
@@ -0,0 +1,107 @@
+package com.patchworkmc.event;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import org.objectweb.asm.Opcodes;
+
+// Currently patchwork cannot handle @SubscribeEvent for overloaded methods
+public class EventSubscriptionChecker {
+	private static class Entry {
+		public List<SubscribeEvent> subscribeEvents;
+		public List<String> superClasses;
+
+		Entry(List<SubscribeEvent> subscribeEvents, List<String> superClasses) {
+			this.subscribeEvents = subscribeEvents;
+			this.superClasses = superClasses;
+		}
+	}
+
+	private Map<String, Entry> entries = new HashMap<>();
+	private static final List<String> missingClassWhiteList = Arrays.asList(
+			"java/",
+			"net/minecraft",
+			"net/minecraftforge"
```

Comment:
A more solid comparison, in case a mod does something like `net.minecraftcomesalive.mod.whatever` or something similar.
```suggestion
			"net/minecraft/",
			"net/minecraftforge/"
```